cli/pkg/kctrl/cmd/package/installed/pause_or_kick.go

```diff
@@ -183,6 +189,7 @@ func (o *PauseOrKickOptions) pause(client kcclient.Interface) error {
 }
 
 func (o *PauseOrKickOptions) unpause(client kcclient.Interface) error {
+	o.statusUI.PrintMessagef("Starting reconciliation for package install '%s' in namespace '%s'", o.Name, o.NamespaceFlags.Name)
```

Comment:
Now that we are printing messages for both pause and unpause, should we remove this line to make it a little less verbose?
```
o.ui.PrintLinef("Triggering reconciliation for package install '%s' in namespace '%s'", o.Name, o.NamespaceFlags.Name)
```

Reply:
This is printed before we ask for confirmation, tells the user what is gonna happen